src/screens/ReplyComment/ReplyCommentId.js

```diff
@@ -327,41 +327,41 @@ const styles = StyleSheet.create({
   containerReply: () => ({
     borderLeftWidth: 1,
     width: '100%'
-    // backgroundColor: 'red',
+    // backgroundColor: COLORS.redalert,
     // flex: 1,
   }),
   seeRepliesContainer: (isLast) => ({
     display: 'flex',
     flexDirection: 'row',
     paddingBottom: 14,
-    borderLeftColor: isLast ? 'transparent' : colors.gray1
+    borderLeftColor: isLast ? 'transparent' : COLORS.lightgrey
   }),
   seeRepliesText: {
-    color: colors.blue
+    color: COLORS.signed_primary
   },
   btn: {
     paddingVertical: 8,
     paddingHorizontal: 20,
     borderRadius: 20
   },
   btnText: {
-    color: '#fff'
+    color: COLORS.white
   },
   headerText: {
     fontFamily: fonts.inter[600],
     fontSize: 14,
     marginLeft: -24,
-    color: '#000',
+    color: COLORS.black,
     alignSelf: 'center'
   },
   image: {
     width: 48,
     height: 48
   },
   input: {
-    backgroundColor: '#F2F2F2',
+    backgroundColor: COLORS.concrete,
     flex: 1,
-    color: '#000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are primarily color updates from the old color scheme to the new one. However, it's important to note that changing colors can have a significant impact on accessibility and user experience. Make sure that the new colors maintain good contrast ratios for readability and meet accessibility standards.



<!-- This is an auto-generated comment by OSS CodeRabbit -->